xc/xc7/archs/artix7_100t/tiles/clk_hrow_bot_r/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+project_ray_tile(
```

Comment:
I believe that CLK_HROW_BOT/TOP can actually be avoided. This because the RR graph is built in a way that it has a direct connection between the BUFHCE's input and its output.

The BUFHCEs will not appear in the architecture file.

Reply:
Agreed